stylix/target.nix

```
@@ -26,9 +29,9 @@ with lib;
       mkEnableOption
       "styling for ${humanName}"
       // {
-        default = config.stylix.autoEnable && autoEnable;
+        default = cfg.enable && cfg.autoEnable && autoEnable;
       }
       // optionalAttrs autoEnable {
-        defaultText = literalExpression "stylix.autoEnable";
+        defaultText = literalExpression "stylix.enable && stylix.autoEnable";
```

Should `defaultText` represent the literal `default` expression:

```suggestion
        defaultText =
          literalExpression
          "stylix.enable && stylix.autoEnable && autoEnable";
```

or

```suggestion
        defaultText =
          literalExpression
          "config.stylix.enable && config.stylix.autoEnable && autoEnable";
```

---

Given that `autoEnable` is internal, and the docs already change based on it's value, I don't see any reason to mention this in the public docs.

---

Should we remove `defaultText`?

---

I mean just the `autoEnable` part, so we should use

```nix
defaultText = literalExpression "config.stylix.enable && config.stylix.autoEnable";
```

---

This appears to already be the current state; I have no strong opinion on whether or not we include the `config` part for now.

Marking this as resolved.